src-java/floodlight-service/floodlight-api/src/main/java/org/openkilda/messaging/command/switches/SwitchRulesInstallRequest.java

```diff
@@ -124,13 +131,4 @@ public void setIslPorts(List<Integer> islPorts) {
     public void setFlowPorts(List<Integer> flowPorts) {
         this.flowPorts = flowPorts;
     }
-
```

Comment:
why this method is removed?

Reply:
Two reasons:
1. It's invalid because it contains only 3 fields but class has much more.
2. To do not add new field into `toString()` method, because it will be generated be @Data annotation